charts/mlp/values.yaml

```diff
@@ -77,7 +77,16 @@ gitlab:
   clientId: #clientId
   clientSecret: #clientSecret
 
+# If you would like to use an external postgres database, you can connect using these credentials
+externalPostgres:
+  enabled: false
+  username: mlp
+  database: mlp
+  password: password
```

Comment:
Is the password stored in secret object?

Reply:
Yes it is stored in the secret object here: https://github.com/gojek/mlp/pull/47/files#diff-7e3c0b3ea7142b17fb9b5d25fa19c6fb0edbccf4b1ca69819f890f4712154075, which mimics the Postgres chart.